api/db/affiliations.js

```diff
@@ -200,9 +201,27 @@ const updateAuthAffiliation = async ({
     throw new Error('User is editing their own affiliation');
   }
 
+  // Lookup role id of the user making the affiliation update
+  const { role_id: changedByRoleId } = await (transaction || db)(
+    'auth_affiliations'
+  )
+    .select('role_id')
+    .where({ state_id: changedByStateId, user_id: changedBy })
+    .first();
+
+  // Lookup role name of who is updating the affiliation
+  const { name: changedByRoleName } = await (transaction || db)('auth_roles')
```

Comment:
you could combine this query and the one above with a join and only have to do one query

Reply:
I actually don't need either of these now since I can just use `req.user.role`